src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -675,32 +680,17 @@ protected function getUTCDateTime(\DateTime $dateTime)
     /**
      * Removes all HTML markup & decode HTML entities
      *
+     * @deprecated
+     *
      * @param $text
      *
      * @return string
      */
     protected function removeHtml($text)
     {
-        if (!trim($text)) {
-            return '';
-        }
-
-        // Remove Styles and Scripts
-        $crawler = new Crawler($text);
-        $crawler->filter('style, script')->each(function (Crawler $crawler) {
-            foreach ($crawler as $node) {
-                $node->parentNode->removeChild($node);
-            }
-        });
-        $text = $crawler->html();
-
-        // Remove HTML markup
-        $result = strip_tags($text);
+        @trigger_error(sprintf('The "%s::%s" method is deprecated in NodeSearchBundle 5.1 and will be removed in NodeSearchBundle 6.0.', __CLASS__,__METHOD__), E_USER_DEPRECATED);
```

Comment:
It's enough to only use the `__METHOD__` constant as the FQCN is included.

Reply:
right you are